src/ChunkDef.h

```diff
@@ -162,6 +162,13 @@ class cChunkDef
 		return AbsoluteToRelative(a_BlockPosition, chunkPos);
 	}
 
+	inline static Vector3i AbsoluteToRelative(const Vector3d & a_BlockPosition)
+	{
+		// pos need floor, then call vec3i overload func
+		// if use default double -> int, will cast -1.xx -> -1(actually need to be -2)
+		return AbsoluteToRelative(a_BlockPosition.Floor());
+	}
+
```

Comment:
I still think this function is not needed at all, all it does is use `.Floor` on the input param, the caller can do that no problem.